src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -383,28 +395,38 @@ def _are_unit_indices_overlapping(sparsity_mask, i, j):
     Returns
     -------
     bool
-        True if the unit indices i and j are overlapping, False otherwise
+        True if the units i and j are spatially overlapping, False otherwise
     """
     if np.any(sparsity_mask[i] & sparsity_mask[j]):
         return True
     else:
         return False
 
 
-def find_collisions(spikes, spikes_w_margin, delta_collision_samples, sparsity_mask):
+def find_collisions(spikes, spikes_within_margin, delta_collision_samples, sparsity_mask):
     """
     Finds the collisions between spikes.
 
+    Given an array of spikes extracted from all units, find the 'spike collisions'
+    - incidents where two spikes from different units overlap temporally and spatially.
+
+    First for each spike, find all other spikes that temporally overlap the spike.
+    Temporal overlap is defined as another spike peak occurring within a time window around
+    the spike peak. Next, only these temporally overlapping spikes that also
+    spatially overlap the spike are included in the output `collision_spikes_dict`.
```

Comment:
temporal overlap is defined, but not spatial.

Reply:
Thanks I changed to
```
    Spatial overlap is defined as the two spikes both having signal on any shared
    channel (i.e. the spikes are not spatially overlapping if they have signal on
    two disjoint sets of channels).
```